WebContent/WEB-INF/views/home.jsp

```diff
@@ -15,6 +15,8 @@
  
 		<input type="text" name="userName" placeholder="Enter your username"><br> 
 		<input type="password" name="password" placeholder="Enter your password"><br>
+		Date of Birth: <input type="date" class="form-control" id="dob"name="dob">
```

Comment:
If you are adding the DOB then please add username and password as well

Reply:
Good idea, thanks for pointing that out. Adding suggested changes